src/app/tests/suites/credentials/TestHarnessDACProvider.cpp

```diff
@@ -106,7 +106,7 @@ constexpr const uint8_t kCdForAllExamples[539] = {
     0x98, 0x96, 0xc7, 0xc9, 0xc8, 0x2d, 0x4b, 0xf5, 0x76, 0xd3, 0x8f, 0xbd, 0x36, 0x5b
 };
 #else  /* Fall back to the VID=0xFFF1 CD */
-constexpr const uint8_t kCdForAllExamples[540] = {
+constexpr const uint8_t kCdForAllExamples[539] = {
```

Comment:
why not just `[]` and let the compiler figure out the size?

Reply:
Maybe just use [] in the other case as well... this manually setting the size seems to be a footgun.

Reply:
I was trying to avoid making waves by not proactively doing that, but I definitely agree. I have a new draft that addresses this issue in most places.